### DISCUSSION TITLE
Is WinDyanmicDesktop project dead?

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [t1m0thyj/WinDynamicDesktop#501] [General]
Hey,
Is WinDyanmicDesktop project dead? There is many bugs to be fixed, and additional features that can be added. and the lastest update we received was on: aug 2022

**Comment by t1m0thyj** [comment]
It's not dead, I just don't have as much time to work on it as I used to. There have been new commits pushed to the main branch as recently as last week in preparation for a new release. Are there specific bugs or features that you think are high priority to address?
